src/main/java/org/craftercms/studio/impl/v2/upgrade/operations/SiteVersionUpgradeOperation.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2007-2018 Crafter Software Corporation. All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.craftercms.studio.impl.v2.upgrade.operations;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.configuration2.Configuration;
+import org.craftercms.studio.api.v2.exception.UpgradeException;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+/**
+ * Implementation of {@link org.craftercms.studio.api.v2.upgrade.UpgradeOperation} that updates or adds the version
+ * file for the given site.
+ * @author joseross
+ */
+public class SiteVersionUpgradeOperation extends XsltFileUpgradeOperation {
```

Comment:
Why extend `XsltFileUpgradeOperation`? It's not doing anything related to XSLT

Reply:
There is an XSTL to update the version tag in any xml file, the difference is that this one adds the file if its missing from the repo.